executor/src/witgen/affine_expression.rs

```diff
@@ -197,7 +195,7 @@ impl AffineExpression {
                 _ => None,
             })
             .flatten()
-            .map(|con| vec![(solve_for.0, Constraint::BitConstraint(con))])
+            .map(|con| EvalValue::complete(vec![(solve_for.0, Constraint::BitConstraint(con))]))
```

Comment:
I'm wondering - can a bit constraint ever be complete? We can use the same polynomial identity to learn about the actual values. Assume we have `X = Y`, and we know a bit constraint on `Y`, but not its value. Then we learn a bit constraint on `X`. By some other means, we learn the value of `Y` and thus can use the same polynomial identity again to learn the value of `X`.